src/transformers/models/bloom/__init__.py

```diff
@@ -14,11 +14,21 @@
 
 from typing import TYPE_CHECKING
 
-from ...utils import OptionalDependencyNotAvailable, _LazyModule, is_tokenizers_available, is_torch_available
+from ...utils import (
+    OptionalDependencyNotAvailable,
+    _LazyModule,
+    is_flax_available,
+    is_tokenizers_available,
+    is_torch_available,
+)
 
 
 _import_structure = {
-    "configuration_bloom": ["BLOOM_PRETRAINED_CONFIG_ARCHIVE_MAP", "BloomConfig", "BloomOnnxConfig"],
+    "configuration_bloom": [
+        "BLOOM_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "BloomConfig",
+        "BloomOnnxConfig",
+    ],
```

Comment:
Not sure why this change? It worked on one line.

Reply:
Interesting! It's back on one-line now: https://github.com/huggingface/transformers/pull/25094/commits/b6d8dc6b76ba1b979fea8045d4ab17e3d0f959c3